docs/3.Custom Policies/YAML Custom Policies.md

```diff
@@ -35,14 +36,27 @@ The possible values for category are:
 * KUBERNETES
 * APPLICATION_SECURITY
 * SUPPLY_CHAIN
-* API_SECURITY
+* API_SECURITY 
+
+The possible values for severity are:
+
+* NONE
```

Comment:
Please adjust according to previous list

Reply:
Oups, Sorry, it's fixed